docs/sql-pyspark-pandas-with-arrow.md

```diff
@@ -165,3 +165,20 @@ Note that a standard UDF (non-Pandas) will load timestamp data as Python datetim
 different than a Pandas timestamp. It is recommended to use Pandas time series functionality when
 working with timestamps in `pandas_udf`s to get the best performance, see
 [here](https://pandas.pydata.org/pandas-docs/stable/timeseries.html) for details.
+
+### Compatibiliy Setting for PyArrow >= 0.15.0 and Spark 2.3.x, 2.4.x
```

Comment:
Shall we not mention EOL versions (2.3.x)? We need to discourage it. Or, at least, we should not advertise it.

Reply:
This is mostly for people that are stuck on an existing Spark version and upgrade pyarrow somehow. They would end up seeing some crazy errors that aren't obvious. Hopefully if they see this, they can fix it instead of making a JIRA, etc. I don't think this is advertising to use an old Spark version, but If you think it's better to remove, I can.